tools/gcs-bucket-mover/gcs_bucket_mover/bucket_details.py

```diff
@@ -51,8 +52,15 @@ def __init__(self, conf, source_bucket):
         # These properties can be skipped with cmd line params, so use the property setters to
         # make the checks
         self.iam_policy = source_bucket.get_iam_policy()
-        self.acl_entities = source_bucket.acl.get_entities()
-        self.default_obj_acl_entities = source_bucket.default_object_acl.get_entities()
+
```

Comment:
Instead of try except, do an explicit check if Uniform bucket level access is enabled or not and set values accordingly.
```suggestion
if source_bucket.iam_configuration.uniform_bucket_level_access_enabled:
   self.acl_entities = None
   self.default_obj_acl_entities = None
else:
  self.acl_entities = source_bucket.acl.get_entities()
  self.default_obj_acl_entities = source_bucket.default_object_acl.get_entities()
```